<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShopInfo</class>
 <widget class="QWidget" name="ShopInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1125</width>
    <height>695</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#ShopInfo{
	background-color: rgb(255, 255, 255);
}
</string>
  </property>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>572</y>
     <width>1101</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>具体操作</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>601</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="3">
      <widget class="QLabel" name="label_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>店铺名称:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="le_id">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="tabletTracking">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
    border-style: outset;
    border-width: 1px;
    border-radius: 5px;

    min-width: 5em;
    padding: 6px;
}</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLineEdit" name="le_name">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
    border-style: outset;
    border-width: 1px;
    border-radius: 5px;

    min-width: 5em;
    padding: 6px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>店铺编号:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>联系方式:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="label_6">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>店铺地址:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLineEdit" name="le_site">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
    border-style: outset;
    border-width: 1px;
    border-radius: 5px;

    min-width: 5em;
    padding: 6px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="le_phone">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
    border-style: outset;
    border-width: 1px;
    border-radius: 5px;

    min-width: 5em;
    padding: 6px;
}</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>20</y>
      <width>221</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QPushButton" name="pb_update">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-image: url(:/ico/pb1.jpg);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 5em;
    padding: 6px;
}
QPushButton:hover
{
	
	background-image: url(:/ico/pb.jpg);
    border-style: inset;
}

QPushButton:pressed{
	
	background-image: url(:/ico/pb2.jpg);
    border-style: inset;
}


</string>
       </property>
       <property name="text">
        <string>修改</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pb_ok">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-image: url(:/ico/pb1.jpg);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 5em;
    padding: 6px;
}
QPushButton:hover
{
	
	background-image: url(:/ico/pb.jpg);
    border-style: inset;
}

QPushButton:pressed{
	
	background-image: url(:/ico/pb2.jpg);
    border-style: inset;
}


</string>
       </property>
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pb_delete">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-image: url(:/ico/pb1.jpg);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 5em;
    padding: 6px;
}
QPushButton:hover
{
	
	background-image: url(:/ico/pb.jpg);
    border-style: inset;
}

QPushButton:pressed{
	
	background-image: url(:/ico/pb2.jpg);
    border-style: inset;
}


</string>
       </property>
       <property name="text">
        <string>注销</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pb_cancel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-image: url(:/ico/pb1.jpg);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 5em;
    padding: 6px;
}
QPushButton:hover
{
	
	background-image: url(:/ico/pb.jpg);
    border-style: inset;
}

QPushButton:pressed{
	
	background-image: url(:/ico/pb2.jpg);
    border-style: inset;
}


</string>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>12</y>
     <width>1100</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>条件查询</string>
   </property>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>671</width>
      <height>35</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>查询条件:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>查询属性:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>全部店铺</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>店铺编号</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>店铺名称</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLineEdit" name="le_select">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
    border-style: outset;
    border-width: 1px;
    border-radius: 5px;

    min-width: 5em;
    padding: 6px;
}</string>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QPushButton" name="pb_select">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-image: url(:/ico/pb1.jpg);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 5em;
    padding: 6px;
}
QPushButton:hover
{
	
	background-image: url(:/ico/pb.jpg);
    border-style: inset;
}

QPushButton:pressed{
	
	background-image: url(:/ico/pb2.jpg);
    border-style: inset;
}


</string>
       </property>
       <property name="text">
        <string>搜索</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>20</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QPushButton" name="pb_break">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-image: url(:/ico/pb1.jpg);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 5em;
    padding: 6px;
}
QPushButton:hover
{
	
	background-image: url(:/ico/pb.jpg);
    border-style: inset;
}

QPushButton:pressed{
	
	background-image: url(:/ico/pb2.jpg);
    border-style: inset;
}


</string>
       </property>
       <property name="text">
        <string>刷新</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pb_ack">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-image: url(:/ico/pb1.jpg);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 5em;
    padding: 6px;
}
QPushButton:hover
{
	
	background-image: url(:/ico/pb.jpg);
    border-style: inset;
}

QPushButton:pressed{
	
	background-image: url(:/ico/pb2.jpg);
    border-style: inset;
}


</string>
       </property>
       <property name="text">
        <string>返回</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>1100</width>
     <height>485</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>查询结果</string>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>30</y>
      <width>1095</width>
      <height>450</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1095</width>
        <height>450</height>
       </rect>
      </property>
      <property name="showDropIndicator" stdset="0">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="lb_id">
      <property name="geometry">
       <rect>
        <x>138</x>
        <y>301</y>
        <width>16</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="lb_name">
      <property name="geometry">
       <rect>
        <x>138</x>
        <y>339</y>
        <width>16</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>648</width>
        <height>450</height>
       </rect>
      </property>
     </widget>
     <widget class="QwtPlot" name="plot">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>30</y>
        <width>320</width>
        <height>400</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
